h/views/api_flags.py

```diff
@@ -22,6 +22,23 @@ def create(context, request):
     return HTTPNoContent()
 
 
+@api_config(route_name='api.flags',
+            request_method='GET',
+            link_name='flag.index',
+            description='List a users flagged annotations for review.',
+            effective_principals=security.Authenticated)
```

Comment:
I agree. In cases where it's okay to reveal the existence of a resource, we should return a 403 instead of a 404. We can do that change API-wide though.